apps/indexer/lib/indexer/prometheus/rpc_response_ets.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Indexer.Prometheus.ResponseETS do
+  @moduledoc """
+  Create and manage an ETS table that keeps the start and end time of json-rpc requests
+  """
+  use GenServer
+
+  def init(arg) do
+    :ets.new(:wrapper, [
+      :bag,
+      :public,
+      :named_table,
+      {:read_concurrency, true},
+      {:write_concurrency, true}
+    ])
+
+    {:ok, arg}
+  end
+
+  def start_link(arg) do
+    GenServer.start_link(__MODULE__, arg, name: __MODULE__)
+  end
+
+  def get do
```

Comment:
given that `put/2` accepts a key and a value, should this not also have key based lookup?